src/storage/storage_backend.rs

```diff
@@ -160,6 +160,11 @@ pub trait StorageBackend<U: Sync + Send + Debug>: Send + Sync + Debug {
     /// [`Metadata`]: ./trait.Metadata.html
     async fn metadata<P: AsRef<Path> + Send + Debug>(&self, user: &Option<U>, path: P) -> Result<Self::Metadata>;
 
+    /// Returns the `Md5` for the given file.
+    ///
+    /// [`Metadata`]: ./trait.Md5.html
+    async fn md5<P: AsRef<Path> + Send + Debug>(&self, user: &Option<U>, path: P) -> Result<String>;
```

Comment:
To not break every storage back-end implementation out there I think we should implement a default implementation in the trait itself.

Reply:
Yes I agree, didn't know how to do that yet..... newbie here :-)